prow/plugins.yaml

```diff
@@ -127,6 +127,13 @@ milestonestatus:
   # curl -H "Authorization: token <token>" https://api.github.com/orgs/<org-name>/teams
   maintainers_id: 2460384
 
+milestone:
+  # You can curl the following endpoint in order to determine the github ID of your team
+  # responsible for maintaining the milestones:
+  # curl -H "Authorization: token <token>" https://api.github.com/orgs/<org-name>/teams
+  maintainers_id: 2460384
```

Comment:
Why is this duplicated? It seems like the `milestonestatus` and `milestone` sections could be combined. It is probably also worth noting that the `maintainers_id` must correspond with `maintainers_team`.

Reply:
what if a repo doesn't want the `/status ` workflow and only wants to be able to add the milestone?

Reply:
Then they only enable the `milestone` plugin and not the `milestonestatus` plugin. The config is separate from where the plugins are enabled (below) so it can be shared between the plugins without requiring that both plugins run.

Reply:
Alright, I'll modify the milestone plugin to use the MilestoneStatus Plugin.  That sound reasonable?

Reply:
I think naming the shared config section `milestone` would probably be better because `milestonestatus` sounds specific to the `milestonestatus` plugin. Its a breaking change to the config, but I doubt any other prow instances are using the `milestonestatus` plugin so it should be fine to do if we add a line that mentions it in the [announcements](https://github.com/kubernetes/test-infra/tree/master/prow#announcements).